tests/php/Benchmark/Lang/Collections/Vector/VectorA.php

```diff
@@ -50,4 +52,115 @@ public function pop(): PersistentVectorInterface
     {
         return $this;
     }
+
+    public function cdr(): void
+    {
+        // TODO: Implement cdr() method.
```

Comment:
Do we want to have those comments even for the test?
If this class is like a "mock", I would suggest cleaning the methods. 😃

Reply:
We might/could/should revome those comments, indeed. I did let the IDE generate the missing methods that were required for the code to simply run 🏃 . But I don't think it's urgent. I created a PR https://github.com/phel-lang/phel-lang/pull/263 with some improvements for the current Benchmark tests that were introduced here. 
I didn't want to commit and pushed them directly here because I wanted @jenshaase to double-check them before. But yeah, the intention is to include them in this branch/PR.

I am sure we could find some places/scenarios where PHPBench might be really useful especially for regressions (even between branches!).

Reply:
This class was just to test an alternative implementation. I think this can be removed complete. The current implementation is the vector is very good.